src/app/clients/edit/[id]/autofill.ts

```diff
@@ -5,7 +5,9 @@ import { BooleanGroup } from "@/components/DataInput/inputHandlerFactories";
 import { processExtraInformation } from "@/common/formatClientsData";
 
 const getNumberAdultsByGender = (family: Schema["families"][], gender: string): number => {
-    return family.filter((member) => member.age === null && member.gender === gender).length;
```

Comment:
can we define a `isChild` function that can tell us if a family member is a child since this logic is also present in line 24? presumably just `member.age === null || member.age >= 16`?